.monorepolint.config.ts

```diff
@@ -32,6 +32,78 @@ glob.sync(path.join(__dirname, "packages", "turf-*")).forEach(pk => {
 
 module.exports = {
   rules: {
-    ":alphabetical-scripts": {}
+    ":alphabetical-scripts": {},
+
+    ":package-entry": [
+      {
+        options: {
+          entries: {
+            main: "dist/js/index.js",
+            types: "dist/js/index.d.ts",
+            files: ["dist"]
+          }
+        },
+        includePackages: TS_PACKAGES
+      },
+      {
+        options: {
+          entries: {
+            main: "dist/js/index.js",
+            module: "dist/es/index.js",
+            types: "index.d.ts",
+            files: ["dist", "index.d.ts"]
+          }
+        },
+        includePackages: JS_PACKAGES
+      }
+    ],
+
+    ":package-script": [
+      {
+        options: {
+          scripts: {
+            bench: "npm-run-all prepare bench:run",
+            "bench:run": "node bench.js",
+            docs: "node ../../scripts/generate-readmes",
+            prepare: "tsc",
+            pretest: "tsc"
+          }
+        },
+        includePackages: TS_PACKAGES
+      },
+      {
+        options: {
+          scripts: {
+            pretest: "rollup -c ../../rollup.config.js",
+            posttest: "node -r esm ../../scripts/validate-es5-dependencies.js"
+          }
+        },
+        includePackages: JS_PACKAGES
+      },
```

Comment:
This will standardise the script part for the plain JS packages.

Is there any reason we don't include `bench` or `docs`?

Reply:
Half oversight, half work in progress. I'd really want to do `includePackages: [...TS_PACKAGES, JS_PACKAGES]` for all of the standard things, but those blocks are sort of missing at the moment (TS still isn't publishing modules)